int8 sensor1IO
int8 sensor2IO
int8 sensor3IO